README.md

```diff
@@ -163,6 +163,7 @@ Environmental variables:
 | `http_buffer_req_body` |  `http` mode only - buffers request body in memory before forwarding upstream to your template's `upstream_url`. Use if your upstream HTTP server does not accept `Transfer-Encoding: chunked`, for example WSGI tends to require this setting. Default: `false`                |
 | `buffer_http`          |  deprecated alias for `http_buffer_req_body`, will be removed in future version    |
 | `static_path`          |  Absolute or relative path to the directory that will be served if `mode="static"` |
+| `function_ready_endpoint` | When non-empty, requests to `/_/ready` will invoke the function handler with this path. This can be used to provide custom readiness logic. When `max_inflight` is set, the concurrency limit is checked first before proxying the request to the function. |
```

Comment:
If this is always going to be a path on top of `upstream_url` (and not an endpoint or whole URL) then I'd like this to be: `ready_path` just like Prometheus has: `metrics_path` in its config file: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config

Reply:
updated